### ISSUE TITLE
TypeError: CSS.supports is not a function at validateInput 

### ISSUE THREAD

**Issue by pauvictorio** [grubersjoe/react-activity-calendar#138]
Hi, I'm getting this error in Next.js 14. Can you help me out?

Here's the stack trace:
```
 ⨯ Internal error: TypeError: CSS.supports is not a function
    at validateInput (./node_modules/.pnpm/react-activity-calendar@2.6.3_react-dom@18.3.1_react@18.3.1__react@18.3.1/node_modules/react-activity-calendar/build/index.js:340:52)
    at createTheme (./node_modules/.pnpm/react-activity-calendar@2.6.3_react-dom@18.3.1_react@18.3.1__react@18.3.1/node_modules/react-activity-calendar/build/index.js:308:9)
    at eval (./node_modules/.pnpm/react-activity-calendar@2.6.3_react-dom@18.3.1_react@18.3.1__react@18.3.1/node_modules/react-activity-calendar/build/index.js:370:19)
```

The heatmap component:
```tsx
'use client'

import React from 'react'

import { useTheme } from 'next-themes'
import ActivityCalendar from 'react-activity-calendar'
import { Tooltip as ReactTooltip } from 'react-tooltip'

import { activities } from '#/data/random-activities'
import { getMonthDayDate } from '@/lib/utils'

import 'react-tooltip/dist/react-tooltip.css'
import '@/styles/heatmap.css'

export default function Heatmap() {
    const { theme } = useTheme()

    return (
        <div className='mx-auto flex flex-col gap-4'>
            <ActivityCalendar
                data={activities}
                colorScheme={theme === 'dark' ? 'dark' : 'light'}
                hideTotalCount={true}
                renderBlock={(block, activity) =>
                    React.cloneElement(block, {
                        'data-tooltip-id': 'react-tooltip',
                        'data-tooltip-html': `${activity.count || 'No'} asks on ${getMonthDayDate(activity.date)}`,
                    })
                }
                showWeekdayLabels={true}
                theme={{
                    light: ['hsl(240, 5%, 96%)', 'hsl(4, 72%, 63%)'],
                    dark: ['hsl(240, 4%, 16%)', 'hsl(4, 72%, 63%)'],
                }}
            />
            <ReactTooltip id='react-tooltip' />
        </div>
    )
}

```

**Comment by pauvictorio** [comment]
I figured it was a just hydration issue. I simply put a `useEffect` and conditionally render the `<ActivityCalendar />` for when the component mounts.

**Comment by grubersjoe** [comment]
Hey. Yes, unfortunately this was broken in Next.js. But I've worked on this in the last day and just now I published `react-activity-calendar@2.7.0` which should fix SSR hydration. At least I tested it successfully with Next.js 14. So you could remove the `useEffect` again.
